src/harness/testcases/WINNF_FT_S_HBT_testcase.py

```diff
@@ -0,0 +1,155 @@
+    device_d = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_d.json')))
+    device_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    devices = [device_d, device_e, device_c]
+    for device in devices:
+        # inject FCC IDs
+        self._sas_admin.InjectFccId({'fccId': device['fccId']})
+    request = {'registrationRequest': devices}
+    response = self._sas.Registration(request)['registrationResponse']
+    cbsd_ids = []
+    for resp in response:
+        cbsd_ids.append(resp['cbsdId'])
+    cbsd_ids.sort(key=lambda cbsdId: cbsdId)
+    del request, response    
+    # create and send grant requests
+    grant_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_c['cbsdId'] = cbsd_ids[0]
+    grant_d = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_d['cbsdId'] = cbsd_ids[1]
+    grant_d['operationParam']['operationFrequencyRange']['lowFrequency'] = 3560
+    grant_d['operationParam']['operationFrequencyRange']['highFrequency'] = 3570
+    grant_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_e['cbsdId'] = cbsd_ids[2]
+    grant_e['operationParam']['operationFrequencyRange']['lowFrequency'] = 3560
+    grant_e['operationParam']['operationFrequencyRange']['highFrequency'] = 3570
+    request = {'grantRequest': [grant_c, grant_d, grant_e]}
+    # send grant request and get response
+    response = self._sas.Grant(request)['grantResponse']
+    self.assertEqual(len(response), 3)       
+    grant_ids = (response[0]['grantId'], response[1]['grantId'], response[2]['grantId'])
+    del request, response
+    # First Heartbeat
+    first_HB_response = self.sendAndAssertSuccessfulHeartBeat(cbsd_ids, grant_ids, [None, None, None])
+    transmitExpireTime = (first_HB_response[0]['transmitExpireTime'], first_HB_response[1]['transmitExpireTime'], first_HB_response[2]['transmitExpireTime'])
+    # STEP 2
+    # load and inject FSS data
+    fss_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'fss_record_c.json')))
+    # Inject Incumbent Activity with Overlapping Frequency of CBSD
+    fss_c['deploymentParam']['operationParam']['operationFrequencyRange']['lowFrequency'] = \
+                             grant_c['operationParam']['operationFrequencyRange']['lowFrequency']
+    fss_c['deploymentParam']['operationParam']['operationFrequencyRange']['highFrequency'] = \
+                             grant_c['operationParam']['operationFrequencyRange']['highFrequency']
+    
+    self._sas_admin.InjectFss({'record': fss_c})   
+    # load and inject GWPZ data
+    gwpz_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'gwpz_record_c.json')))
+    # Inject Incumbent Activity with Overlapping Frequency of CBSD
+    gwpz_c['deploymentParam']['operationParam']['operationFrequencyRange']['lowFrequency'] = \
```

Comment:
Let's choose a range that's more in line with the real data we expect for GWPZs (and you can do it in the file below; there's no need to override the value here).

Reply:
I reset the range of the record to 3650-3700 MHz, which seems ok according to FCC data, right?

Reply:
Agreed -- thanks!